### ISSUE TITLE
nanopolish variants --ploidy parameter not in help text

### ISSUE THREAD

**Issue by scottgigante** [jts/nanopolish#161]
Just a note that the `--ploidy` parameter is not explained in the help text. Is this the ploidy of the reference genome, or the expected ploidy of my sample?

```
% nanopolish variants
variants: a --reads file must be provided
variants: a --genome file must be provided
variants: --ploidy parameter must be provided
variants: a --bam file must be provided

Usage: nanopolish variants [OPTIONS] --reads reads.fa --bam alignments.bam --genome genome.fa
Find SNPs using a signal-level HMM

  -v, --verbose                        display verbose output
      --version                        display version
      --help                           display this help and exit
      --snps                           only call SNPs
      --consensus=FILE                 run in consensus calling mode and write polished sequence to FILE
      --fix-homopolymers               run the experimental homopolymer caller
  -w, --window=STR                     find variants in window STR (format: ctg:start-end)
  -r, --reads=FILE                     the 2D ONT reads are in fasta FILE
  -b, --bam=FILE                       the reads aligned to the reference genome are in bam FILE
  -e, --event-bam=FILE                 the events aligned to the reference genome are in bam FILE
  -g, --genome=FILE                    the reference genome is in FILE
  -o, --outfile=FILE                   write result to FILE [default: stdout]
  -t, --threads=NUM                    use NUM threads (default: 1)
  -m, --min-candidate-frequency=F      extract candidate variants from the aligned reads when the variant frequency is at least F (default 0.2)
  -d, --min-candidate-depth=D          extract candidate variants from the aligned reads when the depth is at least D (default: 20)
  -c, --candidates=VCF                 read variant candidates from VCF, rather than discovering them from aligned reads
  -a, --alternative-basecalls-bam=FILE if an alternative basecaller was used that does not output event annotations
                                       then use basecalled sequences from FILE. The signal-level events will still be taken from the -b bam.
      --calculate-all-support          when making a call, also calculate the support of the 3 other possible bases
      --models-fofn=FILE               read alternative k-mer models from FILE

Report bugs to https://github.com/jts/nanopolish/issues
```

**Comment by jts** [comment]
It is the expected ploidy of your sample. I'll fix this when I'm back in Toronto.

**Comment by scottgigante** [comment]
Okay, brilliant. Thanks!

**Comment by bc2zb** [comment]
Running nanopolish 0.8.1 and the ploidy error still gets thrown, not sure if you just haven't gotten back to toronto yet.

**Comment by jts** [comment]
@bc2zb are you referring to the missing help text?

**Comment by bc2zb** [comment]
@jts yes, help text is still missing

```
~/nanopolish/0.8.1/nanopolish variants -r output.fastq -b output.bam -g genome.fa -o output.test.vcf

variants: --ploidy parameter must be provided

Usage: nanopolish variants [OPTIONS] --reads reads.fa --bam alignments.bam --genome genome.fa
Find SNPs using a signal-level HMM

  -v, --verbose                        display verbose output
      --version                        display version
      --help                           display this help and exit
      --snps                           only call SNPs
      --consensus=FILE                 run in consensus calling mode and write polished sequence to FILE
      --fix-homopolymers               run the experimental homopolymer caller
      --faster                         minimize compute time while slightly reducing consensus accuracy
  -w, --window=STR                     find variants in window STR (format: <chromsome_name>:<start>-<end>)
  -r, --reads=FILE                     the 2D ONT reads are in fasta FILE
  -b, --bam=FILE                       the reads aligned to the reference genome are in bam FILE
  -e, --event-bam=FILE                 the events aligned to the reference genome are in bam FILE
  -g, --genome=FILE                    the reference genome is in FILE
  -o, --outfile=FILE                   write result to FILE [default: stdout]
  -t, --threads=NUM                    use NUM threads (default: 1)
  -m, --min-candidate-frequency=F      extract candidate variants from the aligned reads when the variant frequency is at least F (default 0.2)
  -d, --min-candidate-depth=D          extract candidate variants from the aligned reads when the depth is at least D (default: 20)
  -x, --max-haplotypes=N               consider at most N haplotype combinations (default: 1000)
      --max-rounds=N                   perform N rounds of consensus sequence improvement (default: 50)
  -c, --candidates=VCF                 read variant candidates from VCF, rather than discovering them from aligned reads
  -a, --alternative-basecalls-bam=FILE if an alternative basecaller was used that does not output event annotations
                                       then use basecalled sequences from FILE. The signal-level events will still be taken from the -b bam.
      --calculate-all-support          when making a call, also calculate the support of the 3 other possible bases
      --models-fofn=FILE               read alternative k-mer models from FILE

Report bugs to https://github.com/jts/nanopolish/issues

```

**Comment by jts** [comment]
Fixed in ba74172
